frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -302,7 +302,7 @@ If you'd like to change the order in which dictionaries are queried (and their r
                     local font_size = G_reader_settings:readSetting("dict_font_size") or 20
                     return T(_("Font size (%1)"), font_size)
                 end,
-                callback = function()
+                callback = function(instance)
```

Comment:
Can you call it `touchmenu_instance`, so we don't have to wonder _an instance of what?_ :)